### PR TITLE
Card view vs column view toggle

### PR DESCRIPTION
**Pull request by wonjun-opensource**

### CODE REVIEW

File: packages/frontend/src/pages/HomePage/Selector.tsx [hunk 1]
```diff
@@ -16,12 +16,18 @@ export type SelectorItem<T> = {
 };
 
 type SelectorProps<T> = {
+  classNames?: Record<string, string>;
```
Comment (0xcaff) [inline packages/frontend/src/pages/HomePage/Selector.tsx:19]: please specify the types

File: packages/frontend/src/pages/HomePage/DelegatesContainer.tsx [hunk 2]
```diff
@@ -77,6 +103,39 @@ export function DelegatesContainer({ fragmentKey, variables }: Props) {
...
+      grid-template-columns: 1fr;
+      & a:first-child > div {
+        border-radius: ${theme.spacing["3"]} ${theme.spacing["3"]} 0 0;
+      }
+      @media (max-width: ${theme.maxWidth["6xl"]}) {
+        grid-template-columns: 1fr;
+        justify-content: space-around;
+      }
+
+      @media (max-width: $max_width_md) {
```
Comment (0xcaff) [inline packages/frontend/src/pages/HomePage/DelegatesContainer.tsx:133]: what is `max_width_md`?

File: packages/frontend/src/pages/HomePage/VoterTabular.tsx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+                  <div>delegated to them</div>
+                </VStack>
+              );
+            }
+          })()}
+
+          {(() => {
+            if (!delegate.statement?.summary) {
+              return <div></div>;
```
Comment (0xcaff) [inline packages/frontend/src/pages/HomePage/VoterTabular.tsx:206]: return null instead?

**Comment by yitongzhang** [comment]
Hey @wonjun-opensource this is a great start. I'm still seeing some UI discrepancies vs the mocks. Once you've had a chance to go over these to make sure they match the designs 100%, then you've got the 👍🏽 from me on the design side.

Screen Shot 2022-11-21 at 7 09 13 PM

image

**Comment by 0xcaff** [comment]
moved to here. https://github.com/0xcaff/nouns-agora/pull/34 will let yitong decide how to proceed

**Pull request closed without merging**